tests/sqllogictests/suites/base/11_data_type/11_0006_data_type_decimal

```diff
@@ -112,3 +112,18 @@ query I
 SELECT CAST(-4.56 AS DECIMAL(6, 2)) / CAST(-1.23 AS DECIMAL(6, 2)) AS result;
 ----
 3.69
+
+query I
+SELECT MIN(CAST(-4.56 AS DECIMAL(6, 2)))
```

Comment:
Need tests that scan more than one row.

Reply:
I try to create select union select to replace create table but failed.

Reply:
such as `select max(number::Decimal(6,2)) from numbers(10)`